### PR TITLE
Feature/iteration 5

### PR DESCRIPTION
**Pull request by kelsiebesingeryeh**
Is this a fix or a feature?

Feature

What is the change/fix?

Added functionality so that when a user adds a comment, the comment is saved to localStorage and the comment stays in the respective array on page refresh (doesn't go away). Also added functionality so that when a user adds a comment, they are only allowed to comment once they have inputed text.

Where should the reviewer start?

Top of the file.

How should this be tested?

Open index.html in new window.

Screenshots(if appropriate)

### CODE REVIEW

File: src/comment.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Comment {
```
Comment (eddiezane) [inline src/comment.js:1]: Consider separating out individual from table classes like we discussed.

File: src/main.js [hunk 2]
```diff
@@ -4,15 +4,25 @@ var titleInput = document.querySelector('.title-input');
...
+var saveCommentButton = document.querySelector('.comment-save-button');
+
 
 window.onload = redrawCardsDisplay()
 saveButton.addEventListener('click', saveIdea);
 titleInput.addEventListener('keyup', toggleSaveButton);
 bodyInput.addEventListener('keyup', toggleSaveButton);
 searchInput.addEventListener('keyup', filterCards)
+commentInput.addEventListener('keyup', toggleSaveCommentButton);
+saveCommentButton.addEventListener('click', function (event) {
```
Comment (eddiezane) [inline src/main.js:23]: Is there a reason this is an anonymous function?

File: src/main.js [hunk 3]
```diff
@@ -23,18 +33,32 @@ showFavButton.addEventListener('click', function (event) {
...
     toggleElement(event);
     updateStar(event)
+  } else if (event.target.classList.contains("card-footer")) {
+    inputFields.classList.toggle('hidden');
+    commentForm.classList.toggle('hidden');
+    cardId.innerText = event.target.parentNode.id;
   }
 })
 
+function toggleSaveCommentButton() {
```
Comment (eddiezane) [inline src/main.js:52]: This is almost identical to `toggleSaveButton`. Can you DRY it up?

**Review by eddiezane — COMMENTED** [review]
Looking good!

**Comment by elsafluss** [comment]
Met with my mentor and he gave us some good advice.

**Pull request merged**